### DISCUSSION TITLE
Guest Access. Configuration of Advanced Options. -- config.json

### DISCUSSION THREAD

**Discussion by Creogenesis** [Ylianst/MeshCentral#6233] [Q&A]
Is it possible to enable the clipboard functionality for a **guest invited via a link** using config.json?

Currently, it is possible to take screenshots and send key combinations.

I am also interested in enabling or disabling other tools for remote desktop management. For example, access to process management on the target system.

It would be nice if, when creating a link for a guest, it were possible to choose which tools would be available on their desktop control panel.

As a new feature, it might be possible in the future.

**Comment by si458** [comment]
No

Sadly because the clipboard uses an admin/user websocket to meshcentral and to your remote device, when a guest has a link setup/accessed, the is no admin/user websocket to identify them which means no way to identify the clipboard correctly
It's very complex but the is already a request for this in the github issues
